cypress/integration/deckBuilder/dryRun.js

```diff
@@ -21,9 +21,25 @@ const isMarkedAffordable = $card => {
 
 describe('Deck Builder — Dry-run', () => {
   const CHEAP_DECK =
-    'NU4xLDVOMiw1RjMsNU4zLDVONCw0TjUsNE42LDJONjIsMk42NywyTjY2LDVONjMsNU4xNg=='
+    'NU4xLDVOMiw1RjMsNU4zLDVOMjMsNU40LDVONSw1TjYsNU42Miw1TjY3LDVONjYsNU4xNg=='
```

Comment:
Contained Unhealthy Hysteria and was checked in the basic test to see if cards are playable on turn 1. Replaced with Hunter's Vengeance

Reply:
Removed Hunter's Vengeance ;)